placenames.mss

```diff
@@ -369,6 +369,38 @@
   }
 }
 
+#placenames-small::quarter {
+  [place = 'quarter'] {
+    [zoom >= 14][zoom < 17] {
+      text-name: "[name]";
+      text-fill: @placenames;
+      text-face-name: @book-fonts;
+      text-halo-fill: @standard-halo-fill;
+      text-halo-radius: @standard-halo-radius * 1.5;
+      [zoom >= 14] {
+        text-halo-fill: white;
+        text-size: 11;
+        text-wrap-width: 65; // 5.0 em
```

Comment:
_em_ can be interpreted a short-hand for the current `text-size`, which is here `11`. So:

- If `text-wrap-width: 65` is what you want, than you have _65 ÷ (11 em⁻¹) ≈ 5.9 em_ for the comment.
- If `// 5.0 em` is what you want, than you have _5.0 em × 11 em⁻¹ = 55_ for the “text‑wrap‑width” value.

I would always recommend to start by defining the _em_ values in the comments, and than simply calculate the real values.